<?xml version="1.0"?>
<launch>

	<arg name="js" default="js0"/>


	<!-- joy node -->
	<node respawn="true" pkg="joy" type="joy_node" name="joy" >
		<param name="dev" type="string" value="/dev/input/$(arg js)" />
		<param name="deadzone" value="0.12" />
		<param name="autorepeat_rate" value="10"/>
	</node>


	<node pkg="controller_teleop" type="rmc_teleop.py" name="teleop_node"/>



</launch>